portfolio/src/main/webapp/script.js

```diff
@@ -13,22 +13,50 @@
 // limitations under the License.
 
 /**
-* Insert/remove the class in the element.
-*/
-function toggleClass(element, classAttribute) {
-  if (!element || typeof element.className !== "string") {
-    return;
+ * Check the validity of the parameters.
+ */
+function check(element, classAttribute, functionName) {
```

Comment:
This additional param is actually an interesting approach. It is however not needed because you'll see the name of the calling function in the stack trace printed with the Exception in the console. So you can just remove the param again from the exception text and function.